spec/system/department_admin_manages_newsletters_spec.rb

```diff
@@ -0,0 +1,238 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'Admin manages newsletters', type: :system do
+  let(:organization) { create(:organization) }
+  let(:area) { create(:area, organization: organization) }
+  let(:department_admin) { create(:department_admin, :confirmed, name: 'Sarah Kerrigan', organization: organization, area: area) }
+  let!(:deliverable_users) { create_list(:user, 5, :confirmed, newsletter_notifications_at: Time.current, organization: organization) }
+
+  before do
+    switch_to_host(organization.host)
+    login_as department_admin, scope: :user
+  end
+
+  describe 'creates and previews a newsletter' do
+    it 'allows a newsletter to be created' do
+      visit decidim_admin.newsletters_path
+
+      within '.secondary-nav' do
+        find('.button.new').click
+      end
+
+      within '.new_newsletter' do
+        fill_in_i18n(
+          :newsletter_subject,
+          '#newsletter-subject-tabs',
+          en: 'A fancy newsletter for %{name}',
+          es: 'Un correo electrónico muy chulo para %{name}',
+          ca: 'Un correu electrònic flipant per a %{name}'
+        )
+
+        fill_in_i18n_editor(
+          :newsletter_body,
+          '#newsletter-body-tabs',
+          en: 'Hello %{name}! Relevant content.',
+          es: 'Hola, %{name}! Contenido relevante.',
+          ca: 'Hola, %{name}! Contingut rellevant.'
+        )
+
+        find('*[type=submit]').click
+      end
+
+      expect(page).to have_content('PREVIEW')
+      expect(page).to have_content("A fancy newsletter for #{department_admin.name}")
+    end
+  end
+
+  describe 'previews a newsletter' do
```

Comment:
Specs should be readable and have meaning. They should _specify_ behavior. Does the following have sense?

```ruby
context "with existing newsletter" do
...
it "previews a newsletter"
```

Reply:
Done